src/test/resources/Features/surveillanceComplaints.feature

```diff
@@ -0,0 +1,70 @@
+@Regression
+@complaints
+Feature: Verify complaints can be added
+         OCD-2912-Create a basic complaint, edit a complaint, delete a complaint
+         OCD-2913-Associate a complaint with a listing
+         OCD-2914-Associate a complaint with criteria
+         OCD-2915-Associate a complaint with surveillance
+
+Scenario Outline: Add a complaint
+    Given I'm logged in as "ROLE_ACB"
+    And I am on complaints page
+    When I click Add New Complaint link
+    Then "Add Complaint" form should open to add new complaint
+    When I enter complaint data in fields: "<ONC-ACB>" "<Received Date>" "<Closed Date>" "<ONC-ACB Complaint ID>" "<ONC Complaint ID>" "<Complainant Type>" "<Summary>"
+    And I click save button
+    Then a new complaint with given "<ONC-ACB Complaint ID>" should be added to CHPL 
+Examples:
+    |ONC-ACB|Received Date|Closed Date|ONC-ACB Complaint ID|ONC Complaint ID|Complainant Type|Summary|
+    |Drummond Group|07/12/2019|07/12/2019|ONCACB001|ONC001|Developer|Test Summary|
+    
+Scenario: Edit a complaint
+    Given I'm logged in as "ROLE_ACB"
+    And I am on complaints page
+    When I click Edit complaint for ONC-ACB Complaint ID "ONCACB001"
+    Then "Edit Complaint" form should open
+    When I edit data in ONC Complaint ID field to "TsT0NC-1525"
+    And I click save button on Edit complaint form
+    Then the edits should save and show "TsT0NC-1525" in complaints view table
+    
+Scenario: Delete a complaint
```

Comment:
This is even harder. We can't assume a complaint exists. I wonder if this scenario needs to create a complaint and then delete it...